.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+The issue tracker is a tool to address bugs.
+Please use the #pocoo IRC channel on freenode or Stack Overflow for questions.
```

Comment:
"bugs with Jinja's code" and "for questions about your own code" might be clearer. Possibly link to SO's MCVE page.